tests/builder/test_base.py

```diff
@@ -125,3 +159,79 @@ def count_markers(path, marker='# Edited by rpmlb'):
         edit_file(empty_spec_path, indicator)
         assert empty_spec_path.exists()
         assert count_markers(empty_spec_path) == 1
+
+
+def test_add_macros(macro_spec_path, prepared_macros):
```

Comment:
Did you see my unit test's function name was reviewed by Miro, and those were changed like below lines.
Please change your function names "test_*" referring below lines.
https://github.com/sclorg/rpm-list-builder/blob/master/tests/test_recipe.py#L24-L66